pkg/kops/kops.go

```diff
@@ -24,12 +24,7 @@ func GetRegionFromKopsControlPlane(ctx context.Context, kcp *kcontrolplanev1alph
 		return nil, err
 	}
 
-	if err != nil {
```

Comment:
Don't we need to return the error if it happens?

Reply:
We have already checked the error in the lines above.